### PR TITLE
docs: add user facing docs

### PR DESCRIPTION
**Pull request by polaroi8d**
The first version of dyrector.io documentation. 

Co-Authors @gerimate

### CODE REVIEW

File: docs/dyrector.io-within-your-workflows/use-cases.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Use cases
+
+### TODO: Vakár.mennyi
```
Comment (nandor-magyar) [inline docs/dyrector.io-within-your-workflows/use-cases.md:3]: This won't stay right?

**Review by nandor-magyar — APPROVED** [review]

File: book.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "root": "./docs",
+    "title:": "dyrectorio User facing documentation",
```
Comment (m8vago) [inline book.json:3]: isn't dyrector.io?

**Comment by m8vago** [comment]
docs/.gitbook/assets/product types_dark (1).png is duplicated?

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+dyrector.io is an open-source deployment platform that helps software teams manage releases & deployments easily and efficiently. While non-specialists are enabled to manage these processes in a simplified, self-service manner, specialists can deploy and manage containerized apps faster through the platform. Find out what you can do with dyrector.io!
+
+**You can find dyrector.io's GitHub repository** [**here**](https://github.com/dyrector-io/dyrectorio/)**. Feel free to contribute!**
+
+{% hint style="info" %}
+This documentation is in the works along with our product’s development. In case you’ve got feedback regarding the docs, please reach out to us at [hello@dyrector.io](mailto:hello@dyrector.io). **Contributions are always welcome.**
+{% endhint %}
+
+**If you want to cut to the chase, go to** [**Tutorials**](broken-reference) **to see how you can execute deployments with dyrector.io.**
```
[the remark this answers is not in the document]
Comment (gerimate) [reply docs/README.md:11]: Broken references are resolved. Weird enough the links worked, yet it showed broken-reference.

File: book.json (hunk 2 above)
Comment (polaroi8d) [reply book.json:3]: Sure, nice catch!

File: docs/dyrector.io-within-your-workflows/components.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+# Components
+
+### Node
+
+Nodes are deployment targets. They can be Docker, Kubernetes or on-premises environments; it really depends on your needs. Specifying at least one Node is a basic requirement of both using dyrector.io and deploying your application. For this reason, we suggest this should be the first step you do after the first login. &#x20;
+
+### **Team**
+
+A group of users working on the same project. On your first login you must define a team, to which you can invite your teammates’ profiles. Teams can have multiple Nodes and Registries.&#x20;
```
Comment (m8vago) [inline docs/dyrector.io-within-your-workflows/components.md:16]: `profiles` is not necessary i think

**Review by m8vago — APPROVED** [review]

**Pull request merged**